### ISSUE TITLE
Add loss function

### ISSUE THREAD

**Issue by chriscyyeung** [chriscyyeung/DTC-TensorFlow#1]
- [x] MSE loss
- [x] Binary Cross Entropy
- [x] Dice loss
- [x] Consistency loss
- [x] Supervised loss
- [x] Consistency weight
- [x] Overall loss
